travis-install-wx.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+rm -Rf wxPython-src-2.8.12.1
+wget http://sourceforge.net/projects/wxpython/files/wxPython/2.8.12.1/wxPython-src-2.8.12.1.tar.bz2/download
```

Comment:
I think that the sourceforge problem does not apply here.

- sourceforge is the official repo for wxPython unless this changes we have to download from there.  
- we do not download a bundled binary installer, but a zipped source file. 
- the package is installed in a container and then discarded.